docker-compose-full.yml

```diff
@@ -93,7 +93,7 @@ services:
 
   generic-tooling:
     image: exercism/generic-tooling-image
-    entrypoint: exercism_local_tooling_webserver
+    entrypoint: tooling_webserver
```

Comment:
@iHiD Was adamant previous about everything being namespaced :)

Reply:
Strong opinions loosely held.

Reply:
Hahaha, and it's true this isn't a "general purpose utility" either...